### DISCUSSION TITLE
Ability to shot photos and record video of AR experience under Android disappeared

### DISCUSSION THREAD

**Discussion by vortice3D** [google/model-viewer#2074] [Q&A]
Hi there:

The ability to take photos and record video, of the AR experience, under Android devices, is apparently missing now.

Today, I've tested some web pages when I'm using the model-viewer component, and this previously working functionality is now unavailable.

By the way, iOS version (quick-look ar mode) keeps allowing you to take the photos and record the videos.

Maybe something is broken now in the scene-viewer side?

I'm tested it  with two different Android devices: Motorola Moto G6 and Samsung Galaxy Tab S4 (both ARCore compatible), both running Android 9 and with Google Chrome 88.0.424.152.

**Comment by elalish** [comment]
Are you sure you're in scene-viewer? WebXR mode does not yet have the ability to record (though we're working on it).

**Reply by ti-hardikshah** [reply]
I am using scene-viewer still not able to view record option. is there any code or example ?

**Reply by elalish** [reply]
Can you post a screen recording? And perhaps a repro link? @tpsiaki in case there's a SV regression.
